AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -24,6 +24,10 @@
 !
 hk01.com##div[id^="ig-"][data-title$="app download"]
 tiktok.com##.footer-guide
+||baby-calendar.jp/images/default_web/banner_side_app.png
+baby-calendar.jp##a[href^="https://itunes.apple.com/jp/app"]
+baby-calendar.jp##a[href^="https://play.google.com/store/apps"]
+baby-calendar.jp##.box-side-bana
```

Comment:
Add `baby-calendar.jp##.floating_btn` https://github.com/AdguardTeam/AdguardFilters/issues/112288#issuecomment-1062171751 (need scrolling a bit)
and
`||baby-calendar.jp/img/common/app/news-header1.png`
(`[https://baby-calendar.jp/smilenews/detail/24890](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbaby-calendar.jp%2Fsmilenews%2Fdetail%2F24890)`)

Reply:
Or maybe the previous rule `||baby-calendar.jp/img/common/app/header_` (I moved to mobile_app) can be trimmed down to `||baby-calendar.jp/img/common/app/`.

Reply:
Directly edited